### PR TITLE
Fixed crash

### PR DESCRIPTION
**Pull request by ranaparamveer**
There was a crash when you go to select ringtone again. The default selection is also moved to on post execute.

### CODE REVIEW

File: ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java [hunk 1]
```diff
@@ -354,9 +354,13 @@ public void onLoadComplete(@NonNull HashMap<String, Uri> ringtone) {
         mRingTones.putAll(ringtone);
 
         final String[] itemTitles = mRingTones.keySet().toArray(new String[mRingTones.size()]);
+        int currentSelectionPos = getUriPosition(mRingTones, mCurrentRingTone.second);
+
         mListView.setAdapter(new ArrayAdapter<>(mContext,
-                android.R.layout.select_dialog_singlechoice,
+                android.R.layout.simple_list_item_single_choice,
```
Comment (kevalpatel2106) [inline ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java]: Why do you change the layout?

**Comment by kevalpatel2106** [comment]
@ranaparamveer Thank you for the PR. I reviewed the code. Please see the code review comment.

File: ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java (hunk 1 above)
Comment (ranaparamveer) [reply ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java]: I needed this layout in my code :)
Not of major concern i guess.

Comment (kevalpatel2106) [reply ringtonepicker/src/main/java/com/kevalpatel/ringtonepicker/RingtonePickerDialog.java]: Please revert it to the old `android.R.layout.select_dialog_singlechoice` and after this, I will merge this PR. Everything else looks fine.

**Comment by ranaparamveer** [comment]
Please check now.

**Comment by kevalpatel2106** [comment]
Approved.

**Pull request merged**